test/helpers/pkcs12.c

```diff
@@ -90,9 +90,6 @@ static EVP_PKEY *load_pkey_asn1(const unsigned char *bytes, int len)
     EVP_PKEY *pkey = NULL;
 
     pkey = d2i_AutoPrivateKey(NULL, &bytes, len);
-    if (!TEST_ptr(pkey))
-        goto err;
```

Comment:
The advantage of keeping this is that a nice error message is printed if `pkey` is NULL.

I don't think this should be changed.

Reply:
Sorry, I see now that Matt C already make a note about the error message.